jest_config/__fixtures__/txReceipts.json

```diff
@@ -9,6 +9,7 @@
       "nonce": "0x32",
       "chainId": 3
     },
+    "valueTransfers": [],
```

Comment:
Is this intentional? This json doesn't match the type now. Has `amount` etc but empty valueTransfers

Reply:
moved `asset` and `amount` to conform to type.